test/test-utils/one-inch-version.ts

```diff
@@ -0,0 +1,19 @@
+import { Network } from '@helpers/network'
+
+type OneInchVersion = 'v4.0' | 'v5.0'
```

Comment:
Check `OneInchCall.ts` and see what could be changed ther based on this. I see that this type can be applied there. Actually you can put that logic in that file instead of creating this one. We start having so many files with same domain logic across the app.